src/config.yaml

```diff
@@ -0,0 +1,239 @@
+title: Resonate
+baseURL: /
+languageCode: en-us
+DefaultContentLanguage: en
+Paginate: 10
+googleAnalytics: ''
+enableRobotsTXT: true
+relativeurls: true
+removePathAccents: true
+pygmentsUseClasses: true
+metaDataFormat: yaml
+markup:
+  goldmark:
+    renderer:
+      hardWraps: true
+      unsafe: true
+languages:
```

Comment:
@timotheegoguely Let's break up the language config into seperate locale-specific files using the approach laid out here: https://gohugo.io/getting-started/configuration/#configuration-directory. Otherwise this files going to get very big.